agent_api_rest/postman/ssi-agent.postman_collection.json

```diff
@@ -1661,6 +1708,40 @@
 							},
 							"response": []
 						},
+						{
+							"name": "Show version",
+							"request": {
+								"method": "GET",
+								"header": [],
+								"url": {
+									"raw": "{{HOST}}/version",
+									"host": [
+										"{{HOST}}"
+									],
+									"path": [
+										"version"
+									]
+								}
+							},
+							"response": []
+						},
+						{
+							"name": "Show info",
+							"request": {
+								"method": "GET",
+								"header": [],
+								"url": {
+									"raw": "{{HOST}}/info",
+									"host": [
+										"{{HOST}}"
+									],
+									"path": [
+										"info"
+									]
+								}
+							},
+							"response": []
+						},
```

Comment:
This change moved the "Show version" and "Show info" requests into the `identity/Documents` folder.

Reply:
I think you can move them under line 1502 below the `"Liveness probe"` request :)